pkg/admission/validator/webhook.go

```diff
@@ -29,10 +27,9 @@ func New(mgr manager.Manager) (*extensionswebhook.Webhook, error) {
 	logger.Info("Setting up webhook", "name", Name)
 
 	return extensionswebhook.New(mgr, extensionswebhook.Args{
-		Provider:   metal.Type,
-		Name:       Name,
-		Path:       "/webhooks/validate",
-		Predicates: []predicate.Predicate{extensionspredicate.GardenCoreProviderType(metal.Type)},
```

Comment:
Is there a specific reason for removing this predicate? 

Btw I see AWS provider instead uses ObjectSelector.
https://github.com/gardener/gardener-extension-provider-aws/blob/master/pkg/admission/validator/webhook.go#L44